documentation/modules/source-vm-prerequisites.adoc

```diff
@@ -10,6 +10,12 @@ The following prerequisites apply to all migrations:
 
 * ISO/CDROM disks must be unmounted.
 * Each NIC must contain one IPv4 and/or one IPv6 address.
-* The VM name must contain only lowercase letters (`a-z`), numbers (`0-9`), or hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
-* The VM name must not duplicate the name of a VM in the {virt} environment.
 * The VM operating system must be certified and supported for use as a link:https://access.redhat.com/articles/973163#ocpvirt[guest operating system with {virt}] _and_ for link:https://access.redhat.com/articles/1351473[conversion to KVM with `virt-v2v`].
+* VM names must contain only lowercase letters (`a-z`), numbers (`0-9`), or hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
+* VM names must not duplicate the name of a VM in the {virt} environment.
++
+[NOTE]
+====
+If a VM has a name that does not comply with the rules, {project-full} automatically assigns a new name to the VM that does comply with them.
+This feature allows a migration to proceed smoothly even if someone entered a VM name that does not follow the rules.
```

Comment:
lgmt just one note, maybe worth mentioning what will be the newly generated name, something like:
`The automatically generated name will be in the following format: excluded chars will be removed, upper case letters will be switched to lower case letters and all underscore ('_') will be changed to a dash ('-') instead.`

Reply:
Done, with slight rewrite to fit style guidelines.

Reply:
ack, looking great!